clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ConsumptionTask.java

```diff
@@ -272,6 +273,15 @@ Double getByteRate(PubSubTopicPartition topicPartition) {
     return 0.0D;
   }
 
+  PubSubTopic getDestinationIdentifier(PubSubTopicPartition topicPartition) {
+    ConsumedDataReceiver<List<PubSubMessage<KafkaKey, KafkaMessageEnvelope, Long>>> dataReceiver =
```

Comment:
this could be rewritten as: `dataReceiver == null ? null : dataReceiver.destinationIdentifier();`

Reply:
Sounds good!